sourced/ml/transformers/basic.py

```diff
@@ -182,6 +183,11 @@ def deserialize_uast(self, row: Row):
         if not row[EngineConstants.Columns.Uast]:
             return
         row_dict = row.asDict()
-        row_dict[EngineConstants.Columns.Uast] = [
-            self.parse_uast(uast) for uast in row[EngineConstants.Columns.Uast]]
+        try:
```

Comment:
This is suboptimal: we lose everything if a single UAST request fails. I would rather code a `for` loop and check each `parse_uast` independently with `continue`

Reply:
done.